components/avatar/avatar.css

```diff
@@ -0,0 +1,70 @@
+* {
+    margin: 0;
+    padding: 0;
+    
+}
+.avatar-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 1rem;
+    position: relative;
+    display: inline;
+    font-family: sans-serif;
+}
+.avatar-container > h1{
+    margin: 1rem;
+    font-size: 2rem;
+    font-weight: 900;
+}
+.avatar-container > p{
+    margin: 1rem;
+    font-size: 1rem;
+    font-weight: 300;
+}
+.avatar-container > h3 {
+    margin: 1rem;
+    font-size: 1.3rem;
+    font-weight: 900;
+
+}
+
+.avatar {
+    margin: 1rem;
+    position: relative;
+    display: inline;
+    
+}
+.avatar > img {
+   border-radius: 50%;
+   height: 5em;
+}
+.avatar-square {
+    margin: 1rem;
+    display: inline;
```

Comment:
margin:1rem would give margin only horizontally but not vertically since you've given display-inline , change the display to inline-block